package.json

```diff
@@ -46,7 +46,8 @@
     "test:ui": "npx playwright test --ui",
     "test:ui:codespaces": "npx playwright test --ui-host=0.0.0.0",
     "nearfs:publish-library:create:car": "ipfs-car pack dist/ --output dist.car",
-    "nearfs:publish-library:upload:car": "NODE_ENV=mainnet node ./node_modules/nearfs/scripts/upload-car.js dist.car"
+    "nearfs:publish-library:upload:car": "NODE_ENV=mainnet node ./node_modules/nearfs/scripts/upload-car.js dist.car",
+    "lint": "yarn prettier . --write"
```

Comment:
Just for parity with linters in other projects, we could do this:

```
    "fmt": "prettier --write '**/*.{js,jsx,ts,tsx,json}'",
    "fmt:check": "prettier --check '**/*.{js,jsx,ts,tsx,json}'",
```

It might help us reuse some workflows